### PR TITLE
dird: Fix Copy Job behaviour regarding Archive Jobs

### PR DESCRIPTION
**Pull request by fbergkemper**

### CODE REVIEW

File: systemtests/scripts/functions [hunk 1]
```diff
@@ -301,6 +301,19 @@ check_files_written()
     fi
 }
 
+check_sd_files_written()
+{
+    LOG=$1
+    FILES="$(awk '/SD Files Written:/ { last=$4 } END { print last }' "$LOG")"
+
+    if [[ "$FILES" -lt 1 || -z "$FILES" ]]; then
+        print_debug "ERROR: Expected SD files written > 0, got $FILES."
```
Comment (franku) [inline systemtests/scripts/functions:310]:
```suggestion
        print_debug "ERROR: Expected number of SD files written > 0, got \"$FILES\"."
```

Comment (franku) [reply systemtests/scripts/functions:310]: In case $FILES is empty this makes more sense.

**Review by franku — APPROVED** [review]
One remark on a debug message, otherwise looks good. 
The test runs successful on my Fedora 33. 
Good work!

File: systemtests/scripts/functions (hunk 1 above)
Comment (fbergkemper) [reply systemtests/scripts/functions]: Good suggestions, made the change. Thanks for the review.

**Pull request merged**